.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: ruby
+sudo: 9000
```

Comment:
It switches our builds to the beta Travis builders running on Gogle Compute. They can finish a Bundler build in about 10-11 minutes, which is 2x the speed of the other builders.